### PR TITLE
chore: support Prefix type in ingress path.

### PR DESCRIPTION
**Pull request by b0nete**
Use pathType as Prefix to work with Application Load Balancer.

closes https://github.com/chartmuseum/charts/issues/36

### CODE REVIEW

**Comment by nerdeveloper** [comment]
Hello @b0nete if you are busy, I can pick it up from where you stopped and start working on it. I will be glad to do it. thank you

**Comment by nerdeveloper** [comment]
I am going to take over this PR and submit it for merging this week

**Comment by fede-r1c0** [comment]
@nerdeveloper can you merge your PR [https://github.com/chartmuseum/charts/pull/42](https://github.com/chartmuseum/charts/pull/42) in this branch?

**Comment by fede-r1c0** [comment]
> thanks for the PR! Just needs one small change
> 
> Also, can you bump the chart version to `3.7.0`?

@cbuto chart version was upgraded to 3.7.0

**Comment by nerdeveloper** [comment]
unfortunately, I am sure I can, as I dont have access to this particular branch

**Comment by fede-r1c0** [comment]
> unfortunately, I am sure I can, as I dont have access to this particular branch

sure, @b0nete or @cbuto can you create this PR?

https://github.com/b0nete/charts/compare/b0nete:patch-1...nerdeveloper:patch-1?expand=1

**Comment by cbuto** [comment]
@fede-r1c0 will @b0nete be handling this PR and working on it? if not, lets close it and move the conversation to https://github.com/chartmuseum/charts/pull/42. It will be easier to work on if @b0nete is unavailable.

**Comment by b0nete** [comment]
Sorry @cbuto, i didn't have notifications enabled i and couldn't see it in time.

**Review by cbuto — CHANGES_REQUESTED** [review]
no problem @b0nete! Just a few minor things to address.

You'll also need to resolve the merge conflicts. 

thanks!

File: src/chartmuseum/Chart.yaml [hunk 1]
```diff
@@ -15,4 +15,4 @@ sources:
   - https://github.com/helm/chartmuseum
 maintainers:
   - name: chartmuseum
-    url: https://github.com/chartmuseum
+    url: https://github.com/chartmuseum
```
Comment (cbuto) [inline src/chartmuseum/Chart.yaml:18]:
```suggestion
    url: https://github.com/chartmuseum
   
```

Comment (fede-r1c0) [reply src/chartmuseum/Chart.yaml]: hmm i will review this in more detail in a few minutes

Comment (cbuto) [reply src/chartmuseum/Chart.yaml]: thanks @fede-r1c0, we'll just need to make sure there is a trailing new line at the end of the file and no trailing spaces

Comment (fede-r1c0) [reply src/chartmuseum/Chart.yaml]: done! i fixed it

**Review by cbuto — APPROVED** [review]
looks good! @nerdeveloper @scbizu want to give it a look over?

**Comment by nerdeveloper** [comment]
lgtm! I'm guessing the docs will be updated automatically, right? @cbuto

**Comment by cbuto** [comment]
good point, the docs are a manual process right now. @b0nete can you add `pathType` to the docs in the README? Once that is in place, let’s merge it!

**Comment by fede-r1c0** [comment]
> good point, the docs are a manual process right now. @b0nete can you add `pathType` to the docs in the README? Once that is in place, let’s merge it!

@cbuto can you check the docs changes?

**Comment by cbuto** [comment]
@fede-r1c0 that looks awesome! thanks for taking the time to update those.

**Pull request merged**